README.md

```diff
@@ -1,106 +1,68 @@
-# iob-cache
-
-IOb-cache is a high-performance configurable open-source Verilog cache. If you use or like this repository, please cite the following article:
-
-Roque, J.V.; Lopes, J.D.; Véstias, M.P.; de Sousa, J.T. IOb-Cache: A High-Performance Configurable Open-Source Cache. Algorithms 2021, 14, 218. https://doi.org/10.3390/a14080218 
+# IOb-cache
 
+IOb-cache is a high-performance configurable open-source Verilog cache. 
 IOb-cache supports pipeline architectures, allowing 1 request per clock cycle (read and write).
-
-IOb-cache has both Native (pipelined) and AXI4 back-end interfaces.
-
-The write policy is configurable: either write-through/not-allocate or write-back/allocate.
-
-Configuration supports the number of ways, address-width, cache's word-size (front-end data with), the memory's word-size (back-end data width), the number of lines and words per line, replacement policy (if set associative), and cache-control module (allows performance measurement, cache invalidation and write-through buffer status).
-
-## Organization of the repository
```

Comment:
It is no harm to leave the text up to here

Reply:
I am sorry, it is not clear for me which text that can be kept. Is it the repository organization?

Reply:
Organization of the repository may be deleted. I meant a short intro after referring to the publication.

Reply:
It is already preserved.